apps/frontend/src/app/(app)/today/page.tsx

```diff
@@ -1,45 +1,37 @@
 "use client"
-import * as React from "react"
 
+import * as React from "react"
 import { ShowAgenda } from "@/src/components/atoms/ShowAgenda"
 import { TodayTextArea } from "@/src/components/TodayTextArea"
 import { TodayMeetings } from "@/src/components/TodayMeetings"
 import { DateCycle } from "@/src/components/atoms/Date"
 import { TodayItems } from "@/src/components/TodayItems"
 import { DatePicker } from "@/src/components/DatePicker"
+import usePersistedState from "@/src/hooks/usePersistedState"
 
 const TodayPage: React.FC = () => {
```

Comment:
Try to keep app router clean and let it handle only routing.

Reply:
> Try to keep app router clean and let it handle only routing.

you mean to say remove states from page?

Reply:
Yeah, just put the whole logic inside a component and render that here. Meaning the component will handle all the features and this app router page.tsx will only have to handle the routing.